### DISCUSSION TITLE
Tren - Ultralytics YOLOv8 Dokümanlar

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#7866] [Docs]
# Tren - Ultralytics YOLOv8 Dokümanlar

YOLOv8 modellerini Ultralytics YOLO  ile eğitmek için tek GPU ve çoklu GPU eğitim örnekleri de dahil olmak üzere adım adım kılavuz

https://docs.ultralytics.com/tr/modes/train/

**Comment by prettymaya** [comment]
hi i trained a model using yolov8. i want to improve this model by adding new data. can i retrain the model with only the new data without uploading all the old and new data? if so can you help me with how to do it?

**Reply by prettymaya** [reply]
from ultralytics import YOLO

model = YOLO('best.pt')  

results = model.train(data='myyaml.yaml', epochs=100, imgsz=640)


is this true

**Reply by pderrenger** [reply]
@barutlift yes, your approach is correct. You can continue training your model (`best.pt`) with new data specified in `myyaml.yaml`. Just ensure that `myyaml.yaml` points to your new dataset and that the dataset is correctly formatted. Here's a concise example:

```python
from ultralytics import YOLO

# Load your previously trained model
model = YOLO('best.pt')

# Continue training with the new data
results = model.train(data='myyaml.yaml', epochs=100, imgsz=640)
```

Make sure that `myyaml.yaml` includes the correct paths to your new images and annotations. The model will continue learning from the new data, building upon the knowledge it gained from the previous training.

**Reply by prettymaya** [reply]
so thanks

**Reply by pderrenger** [reply]
@barutlift you're welcome! If you have any more questions or need further assistance, feel free to ask. Good luck with your continued training! 😊🚀
